packages/shared/src/components/sidebar/SidebarBottomSection.tsx

```diff
@@ -45,6 +63,7 @@ export function SidebarBottomSectionSection({
       {props.sidebarExpanded && !optOutWeeklyGoal && (
         <SidebarRankProgress {...props} disableNewRankPopup={showSettings} />
       )}
+      <ChangelogModal isOpen={isChangelogModalActive} />
```

Comment:
Since we dynamically import the modal (loading performance) it's best to also conditionally render this item only when `isChangelogModalActive` is true.

Reply:
Of course 👍